integration/eclipse/src/startup.ts

```diff
@@ -0,0 +1,42 @@
+import { IVY_TYPES, ToolBar } from '@axonivy/process-editor';
+import { EnableInscriptionAction } from '@axonivy/process-editor-inscription';
+import { EnableViewportAction, ShowGridAction, SwitchThemeAction } from '@axonivy/process-editor-protocol';
+import { EnableToolPaletteAction, GLSPActionDispatcher, IDiagramStartup, RequestTypeHintsAction, TYPES } from '@eclipse-glsp/client';
+import { ContainerModule, inject, injectable } from 'inversify';
+import { IvyDiagramOptions } from './di.config';
+
+import { MonacoUtil } from '@axonivy/inscription-core';
+import { MonacoEditorUtil } from '@axonivy/inscription-editor';
+import * as reactMonaco from 'monaco-editor/esm/vs/editor/editor.api';
+import editorWorker from 'monaco-editor/esm/vs/editor/editor.worker?worker';
+import './index.css';
+
+@injectable()
+export class EclipseDiagramStartup implements IDiagramStartup {
+  @inject(GLSPActionDispatcher)
+  protected actionDispatcher: GLSPActionDispatcher;
+
+  @inject(IVY_TYPES.ToolBar)
+  protected toolBar: ToolBar;
+
+  @inject(TYPES.IDiagramOptions)
+  protected options: IvyDiagramOptions;
+
+  async postRequestModel(): Promise<void> {
+    // TODO: Why do we send a custom request type hints actual with a very specific request id?
```

Comment:
Is this a question for me? Because I don't know why this is needed. 🙈

Reply:
Do you see any special handling for that in the server implementation? (`RequestTypeHintsAction` or `SetTypeHintsAction`). If not, I could simply remove it and let's see what happens ;-)

Reply:
I removed it as it does not seem to be required at all.